src/common/core/main.py

```diff
@@ -43,9 +45,11 @@ def ensure_cli_env() -> typing.Generator[None, None, None]:
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings.dev")
 
     # Set up Prometheus' multiprocess mode
-    if "PROMETHEUS_MULTIPROC_DIR" not in os.environ:
-        prometheus_multiproc_dir_name = tempfile.mkdtemp()
-
+    if not env.str("PROMETHEUS_MULTIPROC_DIR", ""):
+        delete = not env.bool("PROMETHEUS_MULTIPROC_DIR_KEEP", False)
```

Comment:
Probably not the best time to be reviewing this at 5.40 on a Friday, but why does this need to be a double negative? It's spinning my noodle! Can't we just do: 

```suggestion
        delete = env.bool("PROMETHEUS_MULTIPROC_DIR_DELETE", True)
```